features/hr.yml

```diff
@@ -1,6 +1,5 @@
-draft_date: 2024-06-05
 name: <hr>
-description: The `<hr>` element TODO.
+description: The `<hr>` element represents a thematic break between paragraph-level elements.
 spec: https://html.spec.whatwg.org/multipage/grouping-content.html#the-hr-element
 group: html
```

Comment:
Not a blocker, but I'm wondering if there's a second group to be had here, perhaps with `<br>` (https://github.com/web-platform-dx/web-features/pull/1364)? I'm struggling to come up with a name though. "Text breaks"?

Reply:
I lean towards keeping them separate, as they have different visual and semantic meanings.